src/Actions/ManagesTags.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace TestMonitor\Clickup\Actions;
+
+use TestMonitor\Clickup\Transforms\TransformsTags;
+
+trait ManagesTags
+{
+    use TransformsTags;
+
+    /**
+     * Get a list of tags for the provided space.
+     *
+     * @throws \TestMonitor\Clickup\Exceptions\InvalidDataException
+     *
+     * @return \TestMonitor\Clickup\Resources\Tag[]
+     */
+    public function tags($spaceId)
```

Comment:
The parameter `$spaceId` is missing from the doc blocks.

Reply:
Fixed as well.